### PR TITLE
Store execTime metric in nanoseconds

### PR DESCRIPTION
**Pull request by ssncferreira**
**What this PR does / why we need it**:

Enhancement: store `execTime` result summary statistic metric in nanoseconds.

**Which issue(s) this PR fixes**:
Fixes https://github.com/grafana/loki/pull/4949#discussion_r777011750

**Special notes for your reviewer**:

**Checklist**
- [x] Documentation added
- [x] Tests updated
- [ ] Add an entry in the `CHANGELOG.md` about the changes.

### CODE REVIEW

File: docs/sources/api/_index.md [hunk 1]
```diff
@@ -939,7 +939,7 @@ The example belows show all possible statistics returned with their respective d
       },
       "summary": {
         "bytesProcessedPerSecond": 0, // Total of bytes processed per second
-        "execTime": 0, // Total execution time in seconds (float)
+        "execTime": 0, // Total execution time in nanoseconds (int)
```
Comment (owen-d) [inline docs/sources/api/_index.md:942]: @ivanahuckova Do you think this will negatively affect Grafana's usage of our statistics?

**Review by owen-d — APPROVED** [review]
This LGTM, but I'd like to see if it'll affect Grafana's usage of this API

File: docs/sources/api/_index.md (hunk 1 above)
Comment (ivanahuckova) [reply docs/sources/api/_index.md:942]: Kind of. When we are processing result's statistic, [we add a unit to the value](https://github.com/grafana/grafana/blob/main/public/app/plugins/datasource/loki/result_transformer.ts#L321), so we can then display it in nice way. E.g: 

**if it is more than 60 sec, we will show 1 minute:**
image

**if it is less than 1 sec, we will show ms**
image

So if we change this, we will need to add logic to check if the value is in `ns` (integer) or `sec`(float) which will be slightly hacky, but possible. 🙂

Comment (ssncferreira) [reply docs/sources/api/_index.md:943]: Thank you @ivanahuckova 🙂 
As already discussed, both statistic time values `execTime` and `queueTime` will be returned by the HTTP API in second as float.

**Comment by ssncferreira** [comment]
Grafana uses the returned statistics from the HTTP API and expects time values to be in second as float. As discussed and in order to avoid subsequent changes to Grafana, this PR will be closed and the conversion of `queueTime` will be handled in PR https://github.com/grafana/loki/pull/5052

**Pull request closed without merging**